### ISSUE TITLE
Error occurring with recogTest

### ISSUE THREAD

**Issue by Alfred-AdMobilize** [matrix-io/matrix-os#81]
Error

```
undefined service name init [ 'engine', undefined, undefined, 'type', undefined, undefined ]
TODO: Should start VES for  recognition
==== Application recogTest started! ==== 
[]
[ 'FACE_RECOGNITION' ]
No Recognition Component Available for Ping
Components: [ 'led' ]
(recogTest) Matrix OS Application Library Loading...

(recogTest) [M]->app { eventType: 'container-ready', app: 'recogTest', pid: 2093 } app-recogTest-message

(recogTest) [M]->app { eventType: 'container-ready', app: 'recogTest', pid: 2093 } app-recogTest-message

(recogTest) [M]->app { eventType: 'container-ready', app: 'recogTest', pid: 2093 } app-recogTest-message
[M]->app { eventType: 'container-ready', app: 'recogTest', pid: 2093 } app-recogTest-message

[]
[ 'FACE_RECOGNITION' ]
No Recognition Component Available for Ping
Components: [ 'led' ]

```

Config

```

# do not touch this line
configVersion: 2

description: recognition demo
keywords: cv
name: recogTest
displayName: Recog
galleryUrl: https://storage.googleapis.com/admobilize-assets/matrix-appstore-images-tmp/app-tn/recog-g.png
imageUrls:
  - https://storage.googleapis.com/admobilize-assets/matrix-appstore-images-tmp/app-tn/recog.png

events:
  - train
  - recog
  - reset
  
services:
  re:
    engine: recognition
    type: face

screens: 
  - - train
  - - recognize
  
widgets:
  train:
    event: train
    control: button
    label: 'Train your Face'
    value: 'Train'
  recognize:
    event: recog
    control: button
    label: Recognize Your Face
    value: Recognize

```

**Comment by gordinh** [comment]
@Alfred-AdMobilize just to be on equal terms here. About this message: `undefined service name init [ 'engine', undefined, undefined, 'type', undefined, undefined ] ` i've made a PR fixing this now. I have debugged the app and mos too. At the app, i talked with Sean and it looks like when triggering the button event for train || recog faces it disconnects from the mxss serve. Can you look if this is happening with you on the matrix dashboard or Iphone mobile app? And if it's happening can you show me the logs?

**Comment by gordinh** [comment]
fixed on #82
